transport/src/main/java/io/netty/channel/nio/NioEventLoop.java

```diff
@@ -356,6 +356,11 @@ public void run() {
         rebuildSelector0();
     }
 
+    @Override
+    public int registeredChannels() {
+        return selector.keys().size() - cancelledKeys;
```

Comment:
@normanmaurer I looked at the [failures in CI](https://ci.netty.io/job/netty-centos6-java11-prb/780/testReport/junit/io.netty.channel.nio/NioEventLoopTest/testChannelsRegistered/) and I think this should fix them. What do you think?